packages/test-kit/src/with-feature.ts

```diff
@@ -149,6 +149,7 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
     const disposeAfterEach = createDisposables();
     const envDebugMode = 'DEBUG' in process.env;
     const debugMode = !!process.env.DEBUG;
+    const browserToRun = process.env.BROWSER as 'chromium' | 'firefox' | 'webkit' | undefined;
```

Comment:
ts will not check it in runtime, check that the value ius a supported browser and throw a nice error message with a list of valid options

Reply:
Done :)